backend/sirius-web-services-api/src/main/java/org/eclipse/sirius/web/services/api/mapper/IStdDeserializerProvider.java

```diff
@@ -21,7 +21,7 @@
  *            The type returned by the deserializer
  * @author gcoutable
  */
-public interface IDeserializerProvider<T> {
+public interface IStdDeserializerProvider<T> {
```

Comment:
`IStandardDeserializerProvider`? We do not use abbreviations in type names anywhere else.

Reply:
Because it provides and instance of Jackson's StdDeserializer